src/Lachain.Core/ValidatorStatus/ValidatorStatusManager.cs

```diff
@@ -486,5 +504,35 @@ public bool IsWithdrawTriggered()
             return ValidatorAttendance.FromBytes(bytes, cycle,
                 indexInCycle >= StakingContract.AttendanceDetectionDuration);
         }
+
+        [MethodImpl(MethodImplOptions.Synchronized)]
```

Comment:
I think we should remove '[MethodImpl(MethodImplOptions.Synchronized)]' because Run() method also has this attribute and it is always running with while loop. So this method will not be triggered from outside I think

Reply:
@sgladkov Can you please suggest on this?

Reply:
Agree